commands/build.go

```diff
@@ -633,12 +628,18 @@ func buildCmd(dockerCli command.Cli, rootOpts *rootOptions, debugConfig *debug.D
 		cobrautil.MarkFlagsExperimental(flags, "root", "detach", "server-config")
 	}
 
+	flags.StringVar(&options.printFunc, "call", "", `Set method for evaluating build ("check", "outline", "targets")`)
```

Comment:
nit: we could use subrequests names from BuildKit instead of hardcoding like: https://github.com/moby/buildkit/blob/03d2f9fc4074ea8bf36cea2db8d0829f5442864b/frontend/subrequests/lint/lint.go#L19

Would probably need to move these consts to another package in https://github.com/moby/buildkit/tree/03d2f9fc4074ea8bf36cea2db8d0829f5442864b/frontend/subrequests